bin/cypress/local

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+set -eou pipefail
+
+source bin/cypress/vars.sh
+
+node_modules/.bin/cypress install
+node_modules/.bin/cypress run \
+  --browser chrome \
+  --headed \
+  --record \
+  --key 23c824d9-b9eb-4aea-88fd-d0bb06a9eb51
```

Comment:
@ksolo  : I would add ` --config video=false` for local runs and remove the `--record --key ....`